lib/registration.pm

```diff
@@ -867,6 +867,11 @@ sub scc_deregistration {
         quit_packagekit;
         wait_for_purge_kernels;
         assert_script_run('SUSEConnect --version');
+        if (get_var('UPGRADE_TARGET_VERSION') == '15-SP3') {
+            # https://bugzilla.suse.com/show_bug.cgi?id=1189543
+            record_soft_failure 'bsc#1189543';
```

Comment:
I think add the bug title in the record_soft_failure will help others to know the bug info easily.